src/Nordea/Components/SortableTable.elm

```diff
@@ -110,38 +128,55 @@ sortableTextHeader config =
         [ Text.textTinyLight
             |> Text.view
                 [ css
-                    [ displayFlex
-                    , textOverflow ellipsis
-                    , overflow hidden
-                    , color Color.eclipse
-                    ]
-                ]
-                [ Html.text config.label
-                , scrollDirectionChevron
+                    ([ displayFlex
+                     , textOverflow ellipsis
+                     , overflow hidden
+                     , color Color.eclipse
+                     ]
+                        ++ config.textCss
+                    )
                 ]
+                (if config.chevronAlignRight then
+                    List.reverse content
+
+                 else
+                    content
+                )
         ]
 
 
-textHeader : { css : List Css.Style, label : String } -> Html msg
-textHeader config =
+sortableTextHeader : { css : List Css.Style, sorting : Maybe Sorting, onClick : msg, label : String } -> Html msg
+sortableTextHeader config =
+    sortableTextHeaderStyled { css = config.css, textCss = [], chevronAlignRight = False, sorting = config.sorting, onClick = config.onClick, label = config.label }
+
+
+textHeaderStyled : { css : List Css.Style, textCss : List Css.Style, label : String } -> Html msg
+textHeaderStyled config =
```

Comment:
Taking in multiple css params makes the abstraction leaky, as it exposes the internal html structure of the component a bit. Can we avoid it by: 
```elm
textHeader : { css : List Css.Style, label : String } -> Html msg
textHeader config =
    Text.textTinyLight
        |> Text.withHtmlTag headerElement
        |> Text.view
            [ Attrs.title config.label
            , css
                ([ displayFlex
                 , textOverflow ellipsis
                 , overflow hidden
                 , whiteSpace noWrap
                 , color Color.eclipse
                 , css (flex (num 1)
                 ]
                    ++ config.css
                )
            ]
            [ Html.text config.label ]
```

(Same for `sortableTextHeaderStyled` above.) 

This way, we also keep the table interface simples with fewer options.

Reply:
Introduce more explicit APIs here 16da6915a2d1c5f60b262231894eb636d3b05769

Do you think the change you suggested is still needed/ Now internal details are not exposed anymore?

Reply:
Not strictly needed, but I think it is still a nice improvement

Reply:
I will do it in separate PR